xpbooster.java

```diff
@@ -50,145 +57,66 @@ public void onStatChanged(StatChanged statChanged) {
 
 		if (skill == Skill.OVERALL) return;
 
-		int realpreviousxp = realXpTracker.get(skill);
-		int fakepreviousxp = fakeXpTracker.get(skill);
+		int realPreviousXp = realXpTracker.get(skill);
+		int fakePreviousXp = fakeXpTracker.get(skill);
 
-		int realcurrentxp = currentXp;
-		int realXpGained = realcurrentxp - realpreviousxp;
-		int fakecurrentxp;
+		int realCurrentXp = currentXp;
+		int realXpGained = realCurrentXp - realPreviousXp;
+		int fakeCurrentXp;
 
 		//For first startup when stats are changed from 0 to your current stat, couldn't be bothered to figure out a
 		//smarter (read less lazy) way of detecting this.
 		if (realXpGained > 1000){
```

Comment:
I can't test this as mentioned in the PR note, but what about `realXpGained == client.getSkillExperience(skill)`?
Rationale is if you gain xp equal to your total xp in a skill, that must be going from 0 to your current xp on the first startup :)